domain/accounts_test.go

```diff
@@ -1,36 +1,101 @@
 package domain
 
 import (
+	"errors"
+	"reflect"
 	"testing"
+	"time"
 )
 
+var person = Account{
+	Name:    "Vanny",
+	Cpf:     "13323332555",
+	Secret:  "dafd33255",
+	Balance: 2500,
+}
+
 func TestCreateAccount(t *testing.T) {
+	type testCase struct {
```

Comment:
Coloque este teste para rodar em paralelo a outros testes. Vai ser mais rápido rodar todos os testes e pode te ajudar pegar bugs de concorrência.
```suggestion
func TestCreateAccount(t *testing.T) {
	t.Parallel()
	
	type testCase struct {
```